config/core/rootfs-configs.yaml

```diff
@@ -52,6 +52,21 @@ rootfs_configs:
     script: "scripts/install-bootrr.sh"
     test_overlay: "overlays/baseline"
 
+  bullseye-audio:
+    rootfs_type: debos
+    debian_release: bullseye
+    arch_list:
+      - amd64
+      - arm64
+      - armhf
```

Comment:
What's wrong with x86?

Reply:
Oh, I just missed it because most of the other tests weren't running on it, but I can add it. Thanks.

Reply:
Do you mean i386?

Reply:
Probably, yeah. That was a month ago.